### PR TITLE
Fixed code conventions

### PR DESCRIPTION
**Pull request by kaine119**
https://codein.withgoogle.com/dashboard/task-instances/5021956311089152/

I'm Kai En, from the above Google Code-in task. I've fixed some spelling mistakes and coding conventions.

### CODE REVIEW

**Comment by robbyoconnor** [comment]
Your branch has conflicts which must be resolved.

**Comment by robbyoconnor** [comment]
Rebase and Squash your commits please.

**Comment by robbyoconnor** [comment]
@mogoodrich, could you have a look at this?

**Comment by mogoodrich** [comment]
@kaine119  @robbyoconnor  annoying, this diff is so big github won't show me the whole thing.  Kai En, can you summarize what you did? Was it just applying the formatter?  Thanks!

**Comment by dkayiwa** [comment]
Looks like so! :)

**Comment by kaine119** [comment]
@mogoodrich: Yes, I did just apply the formatter. I fixed indents, staggered long lines, and added curly braces to some if statements, i.e:

```java
if (condition) {
    // do stuff
}
```
instead of 
```java
if (condition)
    //do stuff
```

Here's a (mostly accurate) list of changes.
[module-htmlformentry changes.txt](https://github.com/openmrs/openmrs-module-htmlformentry/files/66764/module-htmlformentry.changes.txt)

**Comment by mogoodrich** [comment]
@kaine119  thanks for the very detailed explanation!

**Comment by lluismf** [comment]
@ mogoodrich  Weird that github can't show you the diff, which browser are you using?

**Comment by mogoodrich** [comment]
Chrome.  I can see most of it, but it tells me it is too big to show all of it:

**Comment by lluismf** [comment]
ok got the same message with Firefox (I was looking at another link).

**Comment by mogoodrich** [comment]
@kaine119 I just merged this in, but it resulted in compile errors, so I had to revert.  See here:

http://bamboo.pih-emr.org:8085/browse/MIREBALAIS-HFE-467
